mobile/native/src/ln_dlc/node.rs

```diff
@@ -94,15 +94,27 @@ impl Node {
         self.inner.get_blockchain_height()
     }
 
-    pub fn get_wallet_balances(&self) -> Result<Balances> {
-        let on_chain = self.inner.get_on_chain_balance()?.confirmed;
+    pub fn get_wallet_balances(&self) -> Balances {
+        let on_chain = match self.inner.get_on_chain_balance() {
+            Ok(on_chain) => on_chain.confirmed,
+            Err(e) => {
+                tracing::error!("Failed to get onchain balance. {e:#}");
```

Comment:
I'm not sure this is a good idea. The user will have no idea that the sync failed and will only see a balance of 0. Imho we should catch this error in the frontend and at least show the user an error message with which he can come to us

Reply:
Ok, I will think of something.

Reply:
179a336a5aa59c3525ca384f73297cb10bd81a3a